### PR TITLE
Adding solution to assignment one!

### PR DESCRIPTION
**Pull request by actually-not-an-username**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Adding code to solve assignment 1

## What did you learn from the changes you have made?
Hyper-parameter fine tunning for CIFAR-100, Image data augmentation, early stoppers and.learning rate variability.

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
Maybe applying some transfer learning from some pre-trained models like resnet or something similar

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
A lot of internet searching including some error guidance from Gemini (Google Colab)

## How were these changes tested?
Ran in local and in colab

## A reference to a related issue in your repository (if applicable)
N/A

## Checklist
- [x] I can confirm that my changes are working as intended

### CODE REVIEW

File: 02_activities/assignments/assignment_1.ipynb
Comment (Jayaragi10) [inline 02_activities/assignments/assignment_1.ipynb:1]: Complete

Comment (Jayaragi10) [reply 02_activities/assignments/assignment_1.ipynb:1]: Pls submit Assignment 2 by this Saturday

Comment (actually-not-an-username) [reply 02_activities/assignments/assignment_1.ipynb:1]: Hi @Jayaragi10 I just opened the PR for my assignment 2
